web/webkit/src/test/scala/net/liftweb/http/provider/servlet/OfflineRequestSnapshotSpec.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2010-2011 WorldWide Conferencing, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.liftweb.http.provider.servlet
+
+import net.liftweb.http.provider._
+import net.liftweb.mockweb.WebSpec
+import org.mockito.Mockito._
+import org.specs2.mock.Mockito
+
+
+object OfflineRequestSnapshotSpec extends WebSpec with Mockito {
+
+  "A snapshot request should" in {
+    val mockHttpRequest = mock[HTTPRequest]
+    val mockProvider = mock[HTTPProvider]
```

Comment:
Yeah, right you are! `= null` it would have to be.

It's not “right” in the sense that if this were live code it would be a bad idea. Since this is a test, I usually care a lot less.

Think of it this way: with the mock, if someone tried to call something on `context`, things would blow up (because there's no expectation set up for that mock). So in that sense it's the same as if you set a `null` there: interacting with the object in question will cause issues. But, we save ourselves whatever weird plumbing and other magic the mocking framework does, instead favoring straightforward language constructs.